src/pages/HomePage.js

```diff
@@ -0,0 +1,78 @@
+import React, { useRef } from 'react';
+import { useSelector } from 'react-redux';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { FaCaretLeft, FaCaretRight } from 'react-icons/fa';
+import { selectProperties } from '../redux/properties/propertiesSlice';
+import './style/HomePage.css';
+import 'swiper/css';
+import PropertyCard from '../components/PropertyCard';
+
+export default function HomePage() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const HomePage= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/shahadat3669/rent-home-now/assets/57421823/f6c44f1f-fbc8-4809-bc6e-dbaab48ac66e)

That way, your code will meet the latest standards. 💪